libs/remix-ui/run-tab/src/lib/actions/account.ts

```diff
@@ -111,22 +111,25 @@ export const createNewBlockchainAccount = async (plugin: RunTab, dispatch: React
 export const createSmartAccount = async (plugin: RunTab, dispatch: React.Dispatch<any>) => {
   const localStorageKey = 'smartAccounts'
   const PUBLIC_NODE_URL = "https://go.getblock.io/ee42d0a88f314707be11dd799b122cb9"
-  const PIMLICO_API_KEY =''
-  const BUNDLER_URL = `https://api.pimlico.io/v2/sepolia/rpc?apikey=${PIMLICO_API_KEY}`
+  const toAddress = "0xAFdAC33F6F134D46bAbE74d9125F3bf8e8AB3a44" // can be zero address too
```

Comment:
can you put more information about which address is that?

Reply:
added